cloudprem/modules/replicated/charts/replicated/templates/replicated.yaml

```diff
@@ -77,9 +77,11 @@ spec:
         app: replicated
         tier: master
     spec:
+      imagePullSecrets:
+        - name: replicateddockerhubkey
```

Comment:
It's lame that we need that... Just to pull the replicated container images too?

Reply:
IDK if we "need" that. This is the yaml that was generated by the update replicated release shell script provided by them. Something must've changed since the last version.

Reply:
Alright. If that's auto-generated, lets not worry about it. :+1: